IronFoundry.Container/Container.cs

```diff
@@ -137,17 +138,30 @@ public IContainerProcess Run(ProcessSpec spec, IProcessIO io)
                 directory.MapUserPath(spec.ExecutablePath) :
                 spec.ExecutablePath;
 
+            var workingDirectory = spec.WorkingDirectory ?? DefaultWorkingDirectory;
+            workingDirectory = !spec.DisablePathMapping
```

Comment:
We should try to map paths unless absolutely necessary.  See previous comment about handling `@ROOT@` in the Warden.